src/javascript/events/custom.js

```diff
@@ -1,6 +1,18 @@
 import Core from '../core';
 import utils from '../utils';
 
+/**
+ * @typedef {Object} TrackingEvent
+ * @property {EventDetail} detail - The custom o-tracking event details
+ *
+ * @typedef {Event & TrackingEvent} OTrackingEvent
```

Comment:
This is creating a type-definition named OTrackingEvent which is a combination of the Event type-definition and the TrackingEvent type-definition. This is a way in JSDoc to mimic inheritance.

Reply:
interesting